Common/Data/Consolidators/RenkoConsolidator.cs

```diff
@@ -81,12 +66,11 @@ public RenkoConsolidator(decimal barSize, RenkoType type)
         /// <param name="evenBars">When true bar open/close will be a multiple of the barSize</param>
         public RenkoConsolidator(decimal barSize, bool evenBars = true)
         {
+            EpsilonCheck(barSize);
             _barSize = barSize;
             _selector = x => x.Value;
-            _volumeSelector = x => 0;
             _evenBars = evenBars;
-
-            Type = RenkoType.Classic;
+            _volumeSelector = x => 0;
```

Comment:
Believe this constructor could call the one bellow `: this(barSize, x => x.Value, x => 0, evenBars)`

Reply:
I tried that, though it seems to cause a cyclic constructor call

Reply:
Please try `: this(barSize, x => x.Value, x => 0, evenBars)` should work, the other constructor isn't calling no other 👍